fake_switches/dell/command_processor/enabled.py

```diff
@@ -159,6 +162,26 @@ def get_vlan_port_configuration(self, port):
 
         return conf
 
+    def get_bridge_address_table(self):
+        output_lines = [
+            "",
+            "Aging time is 300 Sec"
+            "",
+            "  Vlan        Mac Address       Port          Type",
+            "-------- --------------------- ---------- --------------------",
+            "1201     F8B1.5649.0F99        vlan 1201  Management",
+            "1206     F8B1.5649.0F99        vlan 1206  Management",
+            "3991     02E0.5236.3801        ch1        Dynamic",
+            "3991     748E.F8A7.1B01        ch1        Dynamic",
+            "4063     0013.C601.53A7        1/g22      Dynamic",
+            "4063     00C0.B785.3213        ch1        Dynamic",
+            "4063     00C0.B7B4.742C        ch1        Dynamic",
+            "",
+            "Total MAC Addresses in use:206",
+            "",
+        ]
+        return output_lines
```

Comment:
Could have been generated dynamically with a "render_bridge_address_table" method that takes a list of MacAddress objects. With that, it will be more like how the running-config is generated (line 68 to 76).
But it's not a big deal, the result will be the same.